requirements_test.txt

```diff
@@ -16,3 +16,4 @@ pytest-sugar>=0.7.1
 requests_mock>=1.0
 mock-open>=1.3.1
 flake8-docstrings==1.0.2
+asynctest>=0.8.0
```

Comment:
We already have `pytest-asyncio` and `pytest-aiohttp` that help with async testing.

Please remove this.